docs/reference/modules/uploadfs.md

```diff
@@ -67,13 +96,24 @@ module.exports = {
   </template>
 </AposCodeBlock>
 
+### `disabledFileKey`
+This property takes a string (longer is better) and provides an alternative way to make uploaded assets disabled for access when using local storage. Explicitly preventing web access using the `disable` method blocks local file system access by modifying the file permissions which will lead to problems syncing content with `rsync` or similiar commands. This can be circumvented by using a string passed to the `disabledFileKey` property. This value is used with the filename to create an HMAC key hash that is appended to the filename. This is typically sufficient to obfuscate the file name and prevent access. It is recommended that this option be used from the start of the project, but the module exposes a method, [migrateToDisabledFileKey](#migratetodisabledfilekey), to switch later if desired.
+
+### `uploadsPath`
+By default, this is set to the `/public/uploads` directory at the root of your project by the `@apostrophecms/uploadfs` module. If desired, you can reassign this to a different directory.
+
+### `uploadsUrl`
+By default, this is set to your local apostrophe instance base URL, plus any localization prefix, plus `/uploads` by the `node_modules/apostrophe/modules/@apostrophecms/uploadfs` module. If desired, you can reassign this to a different directory.
```

Comment:
Please do not refer to apostrophe modules by their full filesystem path like this. Just call it `@apostrophecms/uploadfs`, that is its fully qualified name as an apostrophe module, the rules of how that resolves in `node_modules` are a separate topic.

Reply:
Fixed.